ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -0,0 +1,256 @@
+package com.vungle.mediation;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.View;
+import android.widget.RelativeLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.ads.mediation.vungle.VungleInitializer;
+import com.google.android.gms.ads.AdRequest;
+import com.vungle.warren.AdConfig;
+import com.vungle.warren.Banners;
+import com.vungle.warren.LoadAdCallback;
+import com.vungle.warren.PlayAdCallback;
+import com.vungle.warren.Vungle;
+import com.vungle.warren.VungleBanner;
+import com.vungle.warren.VungleNativeAd;
+import com.vungle.warren.error.VungleException;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+class VungleBannerAdapter {
+    private static final String TAG = VungleInterstitialAdapter.TAG;
```

Comment:
minor: wrong tag? It's better to remove or limit logs to errors only anyway.

Reply:
fixed.